src/main/java/com/easypost/model/CreditCard.java

```diff
@@ -0,0 +1,78 @@
+package com.easypost.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.easypost.EasyPost;
+import com.easypost.exception.EasyPostException;
+
+public class CreditCard extends BaseCreditCard {
+    public enum Priority {
+        PRIMARY,
+        SECONDARY
+    }
+
+    /**
+     * Fund a credit card.
+     *
+     * @param primaryOrSecondary primary or secondary payment method.
+     * @param amount amount to fund.
+     * @return CreditCardFund object.
+     * @throws EasyPostException when the request fails.
+     */
+    public static CreditCardFund fund(Priority primaryOrSecondary, int amount) throws EasyPostException {
```

Comment:
Let's swap the param order. Amount first, primaryOrSecondary second (to match Python)